src/BridgedRoom.ts

```diff
@@ -254,7 +254,8 @@ export class BridgedRoom {
 
         // TODO: This only works once from matrix as we are sending the event as the
```

Comment:
irrelevant now?

Reply:
Still relevant for all non-puppeted users.